DeploymentCloud/Deployment.Common/CosmosDB/commons.json

```diff
@@ -883,5 +1053,81 @@
 				}
 			}
 		}
+	},
+	{
+		"name" : "DataXBatchDatabricks",
+		"content" : {
+			"name" : "${sparkJobName}",
+			"cluster" : "$sparkName",
+			"databricksToken" : "${sparkDatabricksToken}",
+			"options" : {
+				"name" : "${sparkJobName}",
+				"new_cluster" : {
+					"spark_version" : "5.3.x-scala2.11",
+					"node_type_id" : "Standard_DS3_v2",
+					"autoscale" : {
+						"min_workers" : "${sparkJobDatabricksMinWorkers}",
+						"max_workers" : "${sparkJobDatabricksMaxWorkers}"
+					},
+					"num_workers" : "${sparkJobDatabricksMinWorkers}",
+					"enableAutoscale" : "${sparkJobDatabricksAutoScale}",
+					"spark_conf" : {
+						"spark\uff0edatabricks\uff0edelta\uff0epreview\uff0eenabled" : true,
+						"spark\uff0esql\uff0ehive\uff0emetastore\uff0eversion" : "1.2.1",
+						"spark\uff0esql\uff0ehive\uff0emetastore\uff0ejars" : "builtin"
+					},
+					"spark_env_vars" : {
+						"DATAX_DEFAULTVAULTNAME" : "$sparkKVName"
+					}
+				},
+				"libraries" : [ 
+					{
+						"jar" : "dbfs:/datax/applicationinsights-core-2.2.1.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/azure-documentdb-1.16.1.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/azure-eventhubs-1.2.1.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/azure-eventhubs-spark_2.11-2.3.6.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/azure-keyvault-webkey-1.1.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/datax-core_2.4_2.11-1.2.0-SNAPSHOT.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/datax-host_2.4_2.11-1.2.0-SNAPSHOT.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/datax-keyvault_2.4_2.11-1.2.0-SNAPSHOT-with-dependencies.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/datax-udf-samples_2.4_2.11-1.2.0-SNAPSHOT.jar"
+					}, 
+					{
+						"jar" : "dbfs:/datax/datax-utility_2.4_2.11-1.2.0-SNAPSHOT.jar"
+					}, 
+					{
```

Comment:
This pattern is in 3 places?  Is there a way to refactor?

Reply:
the design expects the exact content of each cosmos db document. thus cannot set it in a common place